packages/hub/test/scope.test.ts

```diff
@@ -1,206 +1,227 @@
 import { Event, EventHint } from '@sentry/types';
 import { getGlobalObject } from '@sentry/utils';
 
-import { addGlobalEventProcessor, Scope } from '../src';
+import { addGlobalEventProcessor, applyScopeToEvent, cloneScope, Scope, updateScope } from '../src';
+import {
+  addScopeBreadcrumb,
+  addScopeEventProcessor,
+  addScopeListener,
+  clearScope,
+  clearScopeBreadcrumbs,
+  getScopeRequestSession,
+  setScopeContext,
+  setScopeExtra,
+  setScopeExtras,
+  setScopeFingerprint,
+  setScopeLevel,
+  setScopeRequestSession,
+  setScopeSpan,
+  setScopeTag,
+  setScopeTags,
+  setScopeUser,
+  setTransactionName,
+} from '../src/scope';
 
 describe('Scope', () => {
```

Comment:
Removed a lot of `any` and tests are passing ... 🥳 getting out of this mess I created 😄